### PR TITLE
Senegal mock data 2

### PR DESCRIPTION
**Pull request by CHelfgott**
Copied over relevant files.

### CODE REVIEW

File: routes/api/queries/dhis/organizations/senegal.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 const organizationDetails = {
+  useDemoData: true,
   auth: {
     user: SENEGAL_DHIS_USERNAME,
     pass: SENEGAL_DHIS_PASSWORD,
     sendImmediately: false,
   },
   baseURL: 'senegal.dhis2.org/dhis',
-  demographicPairs: [['male', 'female'], ['underAge5', 'age5AndOver']],
+  demographicPairs: [['Male', 'Female'], ['UnderAge5', 'Age5AndOver']],
```
Comment (klandell) [inline routes/api/queries/dhis/organizations/senegal.js]: Why did you capitalize these?

File: routes/api/queries/dhis/organizations/senegal.js [hunk 2]
```diff
@@ -63,7 +64,7 @@ const organizationDetails = {
           schemeLabel: 'Monthly incidence per 1000',
           colors: [
             {
-              hexColor: '#F4C17D',
+              hexColor: '#F7D899',
```
Comment (klandell) [inline routes/api/queries/dhis/organizations/senegal.js]: Why did you change this color?

Comment (CHelfgott) [reply routes/api/queries/dhis/organizations/senegal.js]: Fixed
